recipes/ipysankeywidget/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "ipysankeywidget" %}
+{% set version = "0.2.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1f3a1ab7481ad41c06f05e7524479109d6f5685371a85b800667253221c25fa4
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```

Comment:
You should use the build strong from the example recipe.

Reply:
Thanks, do you mean [this example](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L33): `{{ PYTHON }} -m pip install . -vv`?

I thought I was consistent with [the FAQ](https://github.com/conda-forge/staged-recipes#7-when-or-why-do-i-need-to-use--python---m-pip-install---no-deps---vv): `{{ PYTHON }} -m pip install --no-deps . -vv`